x-pack/plugins/security_solution/server/endpoint/routes/error_handler.ts

```diff
@@ -38,6 +46,10 @@ export const errorHandler = <E extends Error>(
     return res.badRequest({ body: error });
   }
 
+  if (error instanceof EndpointHostNotFoundError) {
```

Comment:
You don't need this (actually: this code will never be reached). `EndpointHostNotFoundError` inherits from `NotFoundError` and there is already a check on line `41` that will return a `notFound()` http response